LICENSE

```diff
@@ -0,0 +1,18 @@
+Copyright ConsenSys Software Inc. 2021. All rights reserved.
```

Comment:
A better default for this repository would be to have no license at all. We can choose a license for individual repositories as we create them.

Reply:
Yeah, I suppose we only need to add a license for the first release of a package. I think there's a form we can use to submit open-sourcing requests to legal. I'll add a todo for myself to check that out.